spec/features/home_spec.rb

```diff
@@ -93,14 +102,51 @@ def login_user
 
     it "selects only relevant cards to display" do
       @test_card = FactoryGirl.create :card, review_date: Date.tomorrow,
+                                             deck_id: deck.id,
                                              user_id: user.id
       expect(page).to have_content "Нет карточек для повторения"
     end
 
     it "Doesn`t show cards of other user" do
       @test_card = FactoryGirl.create :card, review_date: Date.today,
+                                             deck_id: deck.id,
                                              user_id: second_user.id
       expect(page).to have_content "Нет карточек для повторения"
     end
   end
+
+  describe "card selection process with current deck chosen" do
+    let(:alter_deck) { FactoryGirl.create :deck, title: "alter_title",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.